simulator/tabletop.py

```diff
@@ -51,7 +51,7 @@ def __mouse_dragged(self, event):
 
     # Move the cube.
     if (self.__drawngrid == False):
```

Comment:
This is not necessarily something that needs to be fixed, but just as a note: In Python, these kinds of if-statements don't need to be parenthesized.